.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,31 +1,34 @@
-#### Before posting an issue
+---
+name: Bug report
+about: Report a bug
+title: '[BUG]'
+labels: bug
+assignees: ''
 
-Search existing GitHub issues to make sure the issue does not already exist:
-https://github.com/xanaduai/pennylane/issues
+---
 
-If posting a PennyLane issue, delete everything above the dashed line, and fill
-in the template. If the issue is a bug, start the title of the issue with [BUG].
+<!--
+Before posting a bug report
 
-If making a feature request, delete the following template and describe, in detail,
-the feature and why it is needed.
+Search existing GitHub issues to make sure the issue does not already exist:
+https://github.com/xanaduai/pennylane/issues
 
 For general technical details check out our documentation:
 https://pennylane.readthedocs.io
-
--------------------------------------------------------------------------------------------------------------
+-->
```

Comment:
I changed this to a comment instead, meaning that it doesn't have to be deleted by the user before posting.

Reply:
🎉 

I didn't know you could do this!